.github/scripts/setup_spark_remote.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -xv
+
+mkdir -p $HOME/spark
+cd $HOME/spark
+
+hatch shell
+
+version=3.5.1
```

Comment:
make this version to use latest always

Reply:
couldn't find latest in the apache distribution.only stable release versions are available.